internal/gcsx/bucket_manager_test.go

```diff
@@ -116,7 +116,7 @@ func (t *BucketManagerTest) TestSetUpBucketMethodWhenBucketDoesNotExist() {
 	bm.config = bucketConfig
 	bm.gcCtx = ctx
 
-	bucket, err := bm.SetUpBucket(context.Background(), invalidBucketName)
+	bucket, err := bm.SetUpBucket(context.Background(), invalidBucketName, false)
```

Comment:
should we test SetUpBucket with isMultiMount true also ?

Reply:
Good catch. I hadn't thought about updating tests for SetUpBucket. I have now duplicated (with true, false values of isMultiBucketMount) both the tests for SetUpBucket, so there are 4 tests now for it.